exercises/accumulate/AccumulateTest.fs

```diff
@@ -34,4 +34,9 @@ let ``Accumulate reversed strings`` () =
 [<Fact(Skip = "Remove to run test")>]
 let ``Accumulate within accumulate`` () =
     accumulate (fun (x:string) -> String.concat " " (accumulate (fun y -> x + y) ["1"; "2"; "3"])) ["a"; "b"; "c"]
-    |> should equal ["a1 a2 a3"; "b1 b2 b3"; "c1 c2 c3"]
\ No newline at end of file
+    |> should equal ["a1 a2 a3"; "b1 b2 b3"; "c1 c2 c3"]
+
+[<Fact(Skip = "Remove to run test")>]
+let ``Accumulate large data set without stack overflow`` () =
```

Comment:
I like it! The only thing I'd love to see is for the version at the top of the file to be incremented. Could you update that version?

Reply:
No problemo. I incremented the major version from `1.0.0` to `2.0.0`
(_because that's what other people seemed to have done in this repo_ 😸 )